frame/system/src/offchain.rs

```diff
@@ -86,7 +86,11 @@ where
 	/// Submit transaction onchain by providing the call and an optional signature
 	pub fn submit_transaction(
 		call: <T as SendTransactionTypes<LocalCall>>::OverarchingCall,
-		signature: Option<<T::Extrinsic as ExtrinsicT>::SignaturePayload>,
+		signature: Option<(
```

Comment:
I don't get why we have to break down `SignaturePayload`? can't this have `TypeInfo` itself? 

And if it is being broken down and not used, I rather actually delete `SignaturePayload` if it is not being used anymore.